Helper/AuthenticationHelper.php

```diff
@@ -56,10 +55,24 @@ public function getBearerToken(): string
         $token = $this->cache->load(self::CACHE_NAME);
 
         if (empty($token)) {
-            $token = $this->objectManager->create(Authentication::class)->send();
-            $this->cache->save($token, self::CACHE_NAME, [], self::CACHE_TIME);
+            $authenticationData = $this->objectManager->create(Authentication::class)->send();
+            $token = $authenticationData->token;
+            $cacheLifetime = $this->getCacheLifetime($authenticationData->expires_at);
+            $this->cache->save($token, self::CACHE_NAME, [], $cacheLifetime);
         }
 
         return $token;
     }
+
+    /**
+     * @param string $expiresAt
+     * @return int
+     * @throws \Exception
+     */
+    protected function getCacheLifetime(string $expiresAt): int
+    {
+        $expiresAtDate = new \DateTime($expiresAt);
```

Comment:
The $expiresAt string is the response obtained from the Airwallex authentication API, which, according to the documentation, will always return the expire_at variable in the ISO8601 format, which PHP parses without issues.

Example:
```
{
    "expiresAtString": "2022-03-16T11:05:56+0000",
    "expiresAtTimezone": "+00:00",
    "currentString": "2022-03-16T07:45:56+0000",
    "currentTimezone": "+00:00",
    "cacheLifetime": 12000
}
```
Here, expiresAtString is outputted as-is in the $expiresAt variable, and currentString is formatted using `$currentDate->format(\DateTime::ISO8601)`. Cache lifetime is 3h20m because of the use of the test mode for Airwallex, as was clarified to us by Snow in the ticket.

As long as no changes come from yourselves regarding the API response, there should be no issues caused by the missing timezone from the initial DateTime object.